src/color-picker/color-picker.js

```diff
@@ -1,14 +1,17 @@
 import ColorElement from "../common/color-element.js";
 import "../channel-slider/channel-slider.js";
 import "../color-swatch/color-swatch.js";
+import "../space-picker/space-picker.js";
 import * as dom from "../common/dom.js";
 
 const Self = class ColorPicker extends ColorElement {
 	static tagName = "color-picker";
 	static url = import.meta.url;
 	static dependencies = new Set(["channel-slider"]);
+	static globalStyle = new URL("color-picker-global.css", import.meta.url);
 	static shadowStyle = true;
 	static shadowTemplate = `
+		<space-picker id="space_picker" part="space-picker" exportparts="picker: space-picker-base"></space-picker>
```

Comment:
Can we put it in a slot, and listen to input events on the slot? Then people can slot in any arbitrary form element and as long as it produces `input` events, it will work.

Reply:
Done!